packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -65,8 +65,8 @@ exports.sourceNodes = async (
     syncToken,
     spaceId,
     accessToken,
-    environment,
```

Comment:
Is this removal intentional? I think `fetchData` does need `environment` param to work correctly

Reply:
🙈